### PR TITLE
Fpla 2302 deduplication in flux

### PR DESCRIPTION
**Pull request by nkoranne**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/FPLA-2302 

### Change description ###
The flux config files need to be rearranged as per the new helm release. The steps followed are as per https://github.com/hmcts/cnp-flux-config/blob/master/docs/app-deployment.md 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by adusumillipraveen — DISMISSED** [review]
LGTM

File: k8s/namespaces/family-public-law/fpl-case-service/prod.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+      ingressHost: fpl-case-service-prod.service.core-compute-prod.internal
+      environment:
+        RELEASE: NOW
+        IDAM_API_URL: https://idam-api.platform.hmcts.net
+        MANAGE_CASE_UI_BASE_URL: https://manage-case.platform.hmcts.net
+        DOCMOSIS_TORNADO_URL: https://docmosis.platform.hmcts.net
+        GATEWAY_URL: https://gateway.ccd.platform.hmcts.net
+        AUTH_IDAM_CLIENT_BASEURL: https://idam-api.platform.hmcts.net
+        SURVEY_URL_CASESUBMISSION: https://www.smartsurvey.co.uk/s/FamilyPublicLaw?pageurl=application_submission
+        FEES_REGISTER_C2_WITHOUT_NOTICE_KEYWORD: GeneralAppWithoutNotice
```
Comment (tomaszpowroznik) [inline k8s/namespaces/family-public-law/fpl-case-service/prod.yaml:17]: hmm, not related to this PR but it was meant to be temporary, needs to align envs

File: k8s/namespaces/family-public-law/fpl-case-service/perftest.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: fpl-case-service
+  annotations:
+    fluxcd.io/automated: "false"
+    hmcts.github.com/prod-automated: disabled
```
Comment (tomaszpowroznik) [inline k8s/namespaces/family-public-law/fpl-case-service/perftest.yaml:7]: do you know what this one does exactly? was not present in old approach

**Review by tomaszpowroznik — DISMISSED** [review]

File: k8s/namespaces/family-public-law/fpl-case-service/fpl-case-service.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  labels:
+    hmcts.github.com/envInjector: enabled
+    slackChannel: fpla-tech
+  name: fpl-case-service
+  annotations:
```
Comment (adusumillipraveen) [inline k8s/namespaces/family-public-law/fpl-case-service/fpl-case-service.yaml:8]: Dont need thes ehere, they are added by platform by default

Comment (nkoranne) [reply k8s/namespaces/family-public-law/fpl-case-service/fpl-case-service.yaml:8]: can you please let me know which ones are added by platform?

File: k8s/namespaces/family-public-law/fpl-case-service/perftest.yaml (hunk 2 above)
Comment (nkoranne) [reply k8s/namespaces/family-public-law/fpl-case-service/perftest.yaml:7]: after discussion with Praveen, these have been added to keep the perf-test version to a specific prod version number

File: k8s/namespaces/family-public-law/fpl-case-service/prod.yaml (hunk 1 above)
Comment (nkoranne) [reply k8s/namespaces/family-public-law/fpl-case-service/prod.yaml:17]: as agreed, let's get the new structure in place first and then revisit environment specific parameters as part of tech debt

**Review by adusumillipraveen — APPROVED** [review]

**Review by samuelzerozero — APPROVED** [review]

**Pull request merged**